### ISSUE TITLE
Investigate the barriers to upgrading

### ISSUE THREAD

**Issue by niamhW** [browser-update/browser-update#238]
Some research into why users don't upgrade might help.

Possibly users are afraid they will lose their bookmarks or logins etc. If you knew why they won't upgrade, you might be able to offer reassurance and get them onto a platform that updates automatically.

**Comment by josselex** [comment]
Good Idea. We could offer a feedback form where users can easily tell why they don't upgrade in one sentence.

**Comment by hkmaly** [comment]
Don't forget to add the "I already upgraded and you didn't noticed" reason.
#290 #233

**Comment by josselex** [comment]
For the english site I implemented a small feedback button and I'm collecting some answers. I will report on this later.
See here on bottom:
http://browser-update.org/en/update.html?force_outdated=true

**Comment by TPS** [comment]
#294 _could_ be a legitimate expansion to this feedback button.

**Comment by josselex** [comment]
I'm testing it now on the English version page.
http://browser-update.org/en/update.html

2016-10-30 13:16 GMT+01:00 TPS <notifications@github.com>:

**Comment by kaptainkommie** [comment]
This is a good idea. 

I would posit that one of the main reasons browsers are not upgraded, specifically with regards to visitors still using versions of Internet Explorer prior to Edge, is enterprise security policy. I have encountered users at more than one publicly traded global company that stated that their company's security policy enforces use of IE and no other browser. 

After speaking with a system administrator colleague about this, he has suggested that it may be done because it is easier to manage Internet Explorer through Group Policy than Edge and other browsers. While this may actually be due to laziness or uncertainty on the part of that company's security team, the fact that companies with thousands of global users are forcing their employees to continue to use very outdated browsers is a major concern.
